lib/datapacksjob.js

```diff
@@ -763,10 +763,15 @@ DataPacksJob.prototype.buildManifestFromQueries = async function(jobInfo) {
 
                 for (var query of jobInfo.queries) {
                     if (allTypesForQuery.has(query.VlocityDataPackType)) {
+                        if(query.VlocityDataPackType.includes('Version')){
```

Comment:
Should be removable, right?

Reply:
If its not removed, it will leave the trace everywhere in all the jsons exported,  against the field name "VlocityDataPackType"